src/projects/annotations.md

```diff
@@ -1,22 +1,95 @@
 ## Annotations
 
-Comment annotations (See the `ano` command) it's a cross-project metadata feature introduced in radare2 5.9.6.
+Annotations are a powerful feature introduced in Radare2 5.9.6 that allows users to associate cross-project metadata with specific functions. Unlike comments or other metadata that are tied to a specific project or session, annotations are persistent and stored globally. This makes them particularly useful for tracking notes and observations across different projects without needing to worry about manually saving them.
+
+The annotations are associated with each function and stored in a dedicated cache directory, making them available even after you leave the session. You can think of annotations as a place to store function-specific notes, decompilation output, or other important information that you want to keep handy.
+
+### Using the ano
+
+Annotations can be managed using the `ano` command. Below is an overview of the available options:
 
 ```console
 [0x00000000]> ano?
-Usage: ano  [*] # function anotations
+Usage: ano  [*] # function annotations
 | ano            show or edit annotations for the current function
-| ano-*          remove all annotations of current file
-| ano*           dump all annotations in ano= commands
-| ano=[b64text]  set anotation text in base64 for current function
+| ano-*          remove all annotations of the current file
+| ano*           dump all annotations in `ano=` commands
+| ano=[b64text]  set annotation text in base64 for current function
 | anoe           edit annotation
 | anos           show annotation
-| anol           show first line of function annotation if any
+| anol           show the first line of function annotation if any
 [0x00000000]>
 ```
 
-The annotations are not tied to projects or sessions. They are stored in your home as separate files, and they are associated with each function.
+### Key Features
+
+**Persistent Across Sessions:**
+
+Annotations are stored globally in the `~/.local/share/radare2/cache` directory. This ensures that they are accessible across different sessions, even if the project is closed and reopened later.
```

Comment:
Is this somehow configurable/overridable? I'd like to self-contain my project metadata in my own location... if I change between computers I want to have that metadata available on the git repository of MY project.

Reply:
Answering myself: https://book.rada.re/projects/usage.html#configuring-the-project-directory